.eslintrc.js

```diff
@@ -1,17 +1,18 @@
 module.exports = {
   parser: "@typescript-eslint/parser",
-  extends: [
-    "plugin:@typescript-eslint/recommended"
-  ],
+  extends: ["plugin:@typescript-eslint/recommended", "plugin:react/recommended"],
   parserOptions: {
     ecmaVersion: 2018,
-    sourceType: "module"
+    sourceType: "module",
   },
   root: true,
   rules: {
-    "@typescript-eslint/array-type": [
-      "error",
-      { default: "generic" }
-    ]
-  }
+    "@typescript-eslint/array-type": ["error", { default: "generic" }],
+    "react/prop-types": ["off"],
+  },
+  settings: {
+    react: {
+      version: "16.13",
```

Comment:
Using `version: "detect"` doesn't work for us because React isn't installed in the same NPM repo that ESLint is. There may be some way to work around this with workspaces, but I'd like to defer that for now and just hardcode the version.